src/App.tsx

```diff
@@ -29,15 +32,21 @@ const router = createBrowserRouter([
 ]);
 
 const queryClient = new QueryClient();
+export const NotificationContext = React.createContext({});
 
 function App() {
+  const [notification, setNotification] = React.useState({ type: '', message: '' });
```

Comment:
This pollutes readability and breaks SRP. Could you create a separate component called NotificationProvider or something, and put the Context provider, state, and logic for displaying snackbars inside? You can also export from there hook which allows you to use the context, instead of the context itself.

Reply:
![image](https://github.com/broslukasz/react-upskill-basics/assets/14224110/ea9c9603-ec62-4ffe-8e4a-7f12643715d7)

Is it about this parts?

should I transclude this?
![image](https://github.com/broslukasz/react-upskill-basics/assets/14224110/cc713e7c-c288-40f3-8d35-7d06dcf46457)